db/migrate/20201102120213_add_supplier_id_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddSupplierIdToVersions < ActiveRecord::Migration[6.0]
+  def change
+    add_column :versions, :supplier_id, :string
```

Comment:
supplier_id is a uuid - not a string - and probably should have a foreign key constraint. How about:

```
    add_column : versions, : supplier_id, :uuid, null: true
    add_foreign_key :versions, :suppliers, column: :supplier_id
```

Reply:
Ah, I didn't realise there was a special field type for uuids, thanks!